Client/Frontend/Home/HomepageContextMenuHelper.swift

```diff
@@ -182,6 +195,26 @@ class HomepageContextMenuHelper: HomepageContextMenuProtocol {
         }).items
     }
 
+    private func showSendToDevice(site: Site) {
+        typealias sendDelegate = InstructionsViewDelegate & DevicePickerViewControllerDelegate
+        guard let delegate = browserDelegate as? sendDelegate else { return }
```

Comment:
We pass in a `BrowserBarViewDelegate` but then cast this to a `sendDelegate`. This implies the `BrowserDelegate` is something else. The `browserDelegate` should be of type `sendDelegate` already instead of `BrowserBarViewDelegate`.